spectator-api/src/main/java/com/netflix/spectator/api/ActivityCounter.java

```diff
@@ -0,0 +1,65 @@
+package com.netflix.spectator.api;
+
+import java.util.Collections;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * A counter that also keeps track of the time since last update.
+ */
+public final class ActivityCounter implements Counter {
```

Comment:
I wonder if something like `IntervalCounter` would be better at conveying how this is different. Arguably all counters track the amount of activity.